go/teams/get.go

```diff
@@ -24,13 +30,8 @@ func newFinder(g *libkb.GlobalContext) *finder {
 	}
 }
 
-func (f *finder) find(ctx context.Context, g *libkb.GlobalContext, name string) (*Team, error) {
```

Comment:
I added some more functions and shifted things around so I wasn't duplicating code